packages/desktop-client/src/i18n.ts

```diff
@@ -0,0 +1,29 @@
+import { initReactI18next } from 'react-i18next';
+
+import i18n from 'i18next';
+import resourcesToBackend from 'i18next-resources-to-backend';
+
+const loadLanguage = (language: string) => {
+  return import(`./locale/${language}.json`);
+};
+
+i18n
+  .use(initReactI18next)
+  .use(resourcesToBackend(loadLanguage))
+  .init({
+    // While we mark all strings for translations, one can test
+    // it by setting the language in localStorage to their choice.
+    lng: localStorage.getItem('language') || 'cimode',
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏good idea